libs/utils/sidekick.js

```diff
@@ -22,17 +27,37 @@ export default function init({ createTag, loadBlock, loadScript, loadStyle }) {
     getModal(null, { id: 'preflight', content, closeEvent: 'closeModal' });
   };
 
+  const addVersion = async () => {
+    const sk = document.querySelector('helix-sidekick');
+    const statusJson = JSON.parse(sk.getAttribute('status'));
+    const sourceUrl = statusJson?.edit?.url;
+    const sourceCode = sourceUrl?.match(/sourcedoc=([^&]+)/)[1];
+    const sourceId = decodeURIComponent(sourceCode);
+    const url = `https://adobe.sharepoint.com/sites/adobecom/_api/web/GetFileById('${sourceId}')`;
```

Comment:
We should not have specific code for the adobecom site here. There will be other sites:

1. theblog
2. milo-academy
3. helpx
4. stock
5. etc.